Worktez/worktez
# fix(github-integration): fixed ui and logic of github integration

Signed-off-by: Abhishek Mishra <am1426620@gmail.com>

### Functionality:
fixing the UI and logic of Github integration in my-organization and task details page

### Solution:
1. Fixed API trigger for github repo integration.
2. created UI and API  for github PR integration.
3.  created fields in db for projectLink and prLinks.

### Risk level:

### How to test:
step1. in the my-organisation page, try adding a github repo, two three times and also reload the page to see if it is still integrated.
step 2. in the task details page, try adding a pull request, then see if its status is being shown
